### PR TITLE
Delete pax and witnesses

### LINKED ISSUE #133: Delete injured passenger / witness entries
Should be a way to remove accidental / unneeded entries.

Reported by @mboneil10.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #133 

Haven't figured out how to `hide` the delete buttons when it's down to one field.

### CODE REVIEW

**Comment by mboneil10** [comment]
Oh, wow. Didn't even write tests.

**Comment by mboneil10** [comment]
FML ugh

File: coverage/.last_run.json [hunk 1]
```diff
@@ -1,5 +0,0 @@
-{
-  "result": {
-    "covered_percent": 34.94
```
Comment (Anbranin) [inline coverage/.last_run.json]: I don't think you want to commit this bit

Comment (mboneil10) [reply coverage/.last_run.json]: OK whoops

**Comment by mboneil10** [comment]
Closing until tests pass

**Pull request closed without merging**